src/app/questionnaire-item/store/transform-initial-state.ts

```diff
@@ -162,8 +162,9 @@ const getOptionsFromValueSet: (
             valueSets
           )
         : undefined,
-    R.pathOr([], ['compose', 'include', 0, 'concept']),
+    R.pathOr([], ['compose', 'include']),
```

Comment:
The selection of the first element of the array has been removed, correct? Did the format change here?

Reply:
Now we merge all items instead of just selecting the first (I encountered a value set which contained multiple entries in `include`).